crates/shared/src/order_validation.rs

```diff
@@ -510,14 +510,27 @@ impl OrderValidating for OrderValidator {
             .await
             .map_err(ValidationError::Partial)?;
 
+        let verification = match order.from {
+            Some(from) => Some(Verification {
+                from,
```

Comment:
`OrderCreation::from` is documented as 

> If set, the backend enforces that this address matches what is decoded as the signer of the signature. This helps catch errors with invalid signature encodings as the backend might otherwise silently work with an unexpected address that for example does not have any balance.

It doesn't make sense to me, to use it in this way here.

Reply:
Does it not make sense to you that `from` is the address that needs to have the `sell_token` or that we always verify a quote when `from` is set and verified to be the `owner` of the order?
Should I extend the opanapi docs to say that we verify quotes when that field is set?

Reply:
The openapi docs do not mention that this changes anything about quote verification. That is confusing. I'm not sure this concept makes sense to users though. Why not always verify with the order's owner? Would also be good to document `pub verification: Option<Verification>` to explain what impact None vs Some has.

Reply:
Ahh, I didn't notice that order `owner` and `from` don't have to be the same for this kind of request. 🤦‍♂️ 
If `from` is `None` we could still recover the actual `owner` from the signature but the verification logic would not kick in because `from` is `None`.
Good catch!
Will fix that and add more documentation.